docs/account.md

```diff
@@ -330,8 +330,8 @@ This field contains the encrypted password. The encryption is SHA1 and is in the
 
 | Language | Code                                                                |
 |----------|---------------------------------------------------------------------|
-| SQL      | SELECT SHA1(CONCAT(UPPER(\`username\`), ':', UPPER(&lt;pass&gt;))); |
-| PHP      | sha1(strtoupper($username).':'.strtoupper($password));              |
+| SQL      | UPPER(SHA1(CONCAT(UPPER('myusername'), ':', UPPER('mypass'))));     |
+| PHP      | strtoupper(sha1(strtoupper($username).':'.strtoupper($password)));  |
```

Comment:
not sure about the outer `strtoupper`

Reply:
else it will be in lowercase

Reply:
strtoupper at the end of sha1 is useless but not an issue, the server understands both lowercased and uppercased sha1 (even if internally save sha1 in uppercase by default). Such password will be converted in v/s fields for SRP6 protocol needed by client.

this is the same for SQL that is not case sensitive when compares 2 strings, however if you use the same convention in any case, you can avoid to do the uppercase when comparing. So IMHO it's a good change in wiki